cmd/eskip/eskip.go

```diff
@@ -22,7 +22,7 @@ import (
 
 type (
 	command     string
-	commandFunc func(in, out *medium) error
+	commandFunc func(in, out *medium, writeClient *WriteClient) error
```

Comment:
well, i am asking for the reason why is it there. If there is no reason for it, then it shouldn't be there.